becquerel/core/spectrum.py

```diff
@@ -386,6 +463,9 @@ def bin_widths(self):
           UncalibratedError: if spectrum is not calibrated
         """
 
+        warnings.warn('bin_widths is deprecated and will be removed in a '
```

Comment:
Can you raise a [`DeprecationWarning`](https://docs.python.org/3/library/exceptions.html#DeprecationWarning) if someone wants to catch this type of warning.

Reply:
Sure, but it looks like the output is suppressed by default in that case.

Reply:
I have seen that before, I think we want them being shown to everybody, so we should turn them on. Use:

```
warnings.simplefilter('always', DeprecationWarning)
```

In the header right after the import, we should fix that in other places too.

Reply:
It looks like there is also an entry in `setup.cfg` to ignore deprecation warnings:

```
setup.cfg:    ignore:.*getargspec.*:DeprecationWarning
```

Does this only affect submodules or this module as well?

Reply:
I'm not familiar with how the `setup.cfg` file works, but adding the `simplefilter` line above works well. Let me know if I should remove the line from `setup.cfg` too.